movie-browser/src/components/FeatureMovie/FeatureMovie.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const FeatureMovies = () => {
+	return <div>Display Feature Movie Section</div>;
```

Comment:
you should always wrap the body of the `return` in parenthesis.

```
return (
   <div>Display Feature Movie Section</div>
);
```

Reply:
Makes sense, this is due to auto formatting by the editor for single line returns.